projects/templatetags/projects_tags.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from articles.models import ArticlePage
+
+register = template.Library()
+
+
+@register.assignment_tag()
+def project_articles(project_page):
```

Comment:
Would it work to instead have a method on the ProjectPage model which does something like this:

```
def get_articles(self):
    return self.articlepage_set.live().order_by("-first_published_at")
```

Technically we need to grab both article pages and series pages, and thus it might be worth sorting them in Python so we can mix them by first_publsihed_at. You can turn query sets into lists and then merge them and then do something like `reversed(sorted(articles, key='first_published_at'))` Though the value of key might need to be something slightly different. Google Python sort by attribute.